src/QOwnNotes.pro

```diff
@@ -125,6 +125,7 @@ SOURCES += main.cpp\
     helpers/fakevimproxy.cpp \
     helpers/flowlayout.cpp \
     services/databaseservice.cpp \
+    threads/scriptthread.cpp \
```

Comment:
this also needs to go into the cmakefile

Reply:
...and to the test project file

Reply:
`cmakelist.txt`?

Reply:
yes

Reply:
There are multiple `test.pro` files, but I don't think those are what you mean?

Reply:
`tests/unit_tests/testcases/app/app-tests.pri` and `src/CMakeLists.txt`